python/tests/integration/arcticdb/test_arctic.py

```diff
@@ -137,12 +140,30 @@ def test_separation_between_libraries_with_prefixes(moto_s3_uri_incl_bucket):
     assert ac_mars["pytest_test_lib"].list_symbols() == ["test_2"]
 
 
-def test_library_management_path_prefix(moto_s3_uri_incl_bucket, boto_client):
-    test_bucket = sorted(boto_client.list_buckets()["Buckets"], key=lambda bucket_meta: bucket_meta["CreationDate"])[
-        -1
-    ]["Name"]
+@pytest.mark.parametrize(
+    "connection_string, client",
+    [
+        ("moto_s3_uri_incl_bucket", "boto_client"),
+        pytest.param(
+            "azurite_azure_uri_incl_bucket",
+            "azure_client",
+            marks=pytest.mark.skipif(sys.platform != "linux", reason="Pending Azure Storge Windows support"),
+        ),
```

Comment:
This `pytest.param(...)` is quite long and used twice, so might want to extract that to a variable.

Reply:
Closed per discussion with Qi. It will be addressed in bulk in a separate PR.